skymp5-functions-lib/src/utils/persistentStorage.ts

```diff
@@ -10,6 +11,40 @@ export class PersistentStorage {
     return PersistentStorage.instance;
   }
 
+  get rounds(): SweetPieRound[] {
+    const str = mp.readDataFile('persistentStorage.json');
+    const obj = this.parse(str);
+    if (Array.isArray(obj['rounds'])) {
+      if (obj['rounds'].filter((x) => typeof x === 'object').length === obj['rounds'].length) {
```

Comment:
Probably will look good if we join these ifs

And probably `!obj.find((x) => typeof x !== 'object')`

Reply:
Done